tests/ballerina-integration-test-utils/src/main/java/org/ballerinalang/test/agent/server/Request.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.test.agent.server;
+
+import io.netty.handler.codec.http.FullHttpRequest;
+
+import java.nio.charset.StandardCharsets;
+
+/**
+ * The Request class provides convenience helpers to the underyling
+ * HTTP Request.
```

Comment:
Shall we add @ since annotation for all the new classes?

Reply:
done